src/components/src/B2cHomePage/b2c.home.page.tsx

```diff
@@ -27,19 +27,19 @@ import Config from '../../../ep.config.json';
 
 import './b2c.home.page.scss';
 
-import bannerImage1 from '../../../images/site-images/b2c-banner-1.png';
-import bannerImage2 from '../../../images/site-images/b2c-banner-2.png';
-import bannerImage3 from '../../../images/site-images/b2c-banner-3.png';
-import productImage1 from '../../../images/site-images/b2c-product-1.png';
-import productImage2 from '../../../images/site-images/b2c-product-2.png';
-import productImage3 from '../../../images/site-images/b2c-product-3.png';
-import productImage4 from '../../../images/site-images/b2c-product-4.png';
-import productImage5 from '../../../images/site-images/b2c-product-5.png';
-import productImage6 from '../../../images/site-images/b2c-product-6.png';
-import productImage7 from '../../../images/site-images/b2c-product-7.png';
+import bannerImage1 from '../../../images/site-images/optimized/b2c-banner-1.jpg';
```

Comment:
We shouldn't need to rename the file extension. ImageContainer should be fetching the allowed image types from the config, and displaying the fall-back filetypes if the preceding doesn't exist.

Reply:
If we do decide to go with jpgs though our fallback imgs should be changed to jpgs

Reply:
I agree, this was more of a POC to demo different image compression ratios.